pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "cloudpipeline"
+version = "0.1.0"
+description = "Generic Pipeline Mechanism"
+authors = [
+    "Adam Frank <afrank@mozilla.com>",
+    "Bryan Sieber <bsieber@mozilla.com>",
+    "Chris Valaas <cvalaas@mozilla.com>",
+    "Jorge Spiropulo <jspiropulo@mozilla.com>"
+]
+packages = [
+    { include = "pipeline" }
+]
+
+[[tool.poetry.source]]
+name = "mozilla-it"
+url = "https://dp2-prod.appspot.com/pypi"
+
+[tool.poetry.dependencies]
+python = ">3.6"
+tox = "*"
```

Comment:
should this be a dev-dep?

Reply:
It's a bit of a chicken and the egg problem in how we're testing with the docker; I imagine there's a better approach. 

But all dev deps can be listed in tox, so I don't know if anything should be in the dev-deps section.

Reply:
This was also partially the reason I began to look into multiple docker files

Reply:
(Trying a few things out with "extras" but if you get any luck with another approach or direction let me know!)